### PR TITLE
feat: add job manager conventions

### PR DESCRIPTION
**Pull request by razbroc**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: src/semanticConventions/infra.json [hunk 1]
```diff
@@ -37,6 +37,16 @@
             "propertyName": "mapcolonies.infra.job_management.service_url",
             "deprecated": false,
             "description": "API url of job management service"
+          },
+          "taskIds": {
+            "propertyName": "mapcolonies.infra.job_management.task_ids",
+            "deprecated": false,
+            "description": "Array of tasks (uuid array)"
+          },
+          "status": {
```
Comment (asafMasa) [inline src/semanticConventions/infra.json:46]: do you think jobStatus is better?

File: src/semanticConventions/infra.json [hunk 2]
```diff
@@ -37,6 +37,16 @@
...
+            "propertyName": "mapcolonies.infra.job_management.status",
+            "deprecated": false,
+            "description": "Status of job as stored on db (COMPLETED/ IN-PROGRESS/ PENDING)"
```
Comment (asafMasa) [inline src/semanticConventions/infra.json:49]: (COMPLETED/ IN-PROGRESS/ PENDING, etc.)

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

File: src/semanticConventions/infra.json [hunk 3]
```diff
@@ -37,6 +37,16 @@
...
+          },
+          "taskIds": {
+            "propertyName": "mapcolonies.infra.job_management.task_ids",
+            "deprecated": false,
+            "description": "Array of tasks (uuid array)"
+          },
+          "status": {
+            "propertyName": "mapcolonies.infra.job_management.status",
+            "deprecated": false,
+            "description": "Status of job/task as stored on db (COMPLETED/ IN-PROGRESS/ PENDING, etc.)"
```
Comment (shimoncohen) [inline src/semanticConventions/infra.json:49]:
```suggestion
            "description": "Status of job/task as stored on db (COMPLETED, IN-PROGRESS, PENDING, etc.)"
```

File: src/semanticConventions/infra.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
           "jobId": {
             "propertyName": "mapcolonies.infra.job_management.job_id",
             "deprecated": false,
-            "description": "Type of task as stored on db (uuid)"
+            "description": "Type of job as stored on db (uuid)"
```
Comment (shimoncohen) [inline src/semanticConventions/infra.json:19]: This is not a type but an ID. Please change description accordingly.
This comment is relevant for `taskId`, `jobType`, and `taskType`'s desriptions as well.

File: src/semanticConventions/infra.json [hunk 5]
```diff
@@ -37,6 +37,16 @@
             "propertyName": "mapcolonies.infra.job_management.service_url",
             "deprecated": false,
             "description": "API url of job management service"
+          },
+          "taskIds": {
+            "propertyName": "mapcolonies.infra.job_management.task_ids",
+            "deprecated": false,
+            "description": "Array of tasks (uuid array)"
```
Comment (shimoncohen) [inline src/semanticConventions/infra.json:44]:
```suggestion
            "description": "Array of task ids (uuid array)"
```

**Review by shimoncohen — CHANGES_REQUESTED** [review]

File: src/semanticConventions/infra.json (hunk 4 above)
Comment (CptSchnitz) [reply src/semanticConventions/infra.json:19]: descriptions*

File: src/semanticConventions/infra.json (hunk 1 above)
Comment (CptSchnitz) [reply src/semanticConventions/infra.json:46]: even if its like this in the DB, it might be confusing with other statuses across mapcolonies

**Review by CptSchnitz — APPROVED** [review]

**Pull request closed without merging**